web/app/view/dialog/ReportConfigController.js

```diff
@@ -26,23 +26,25 @@ Ext.define('Traccar.view.dialog.ReportConfigController', {
     ],
 
     onSaveClick: function (button) {
-        var eventType;
-        this.getView().callingPanel.deviceId = this.lookupReference('deviceField').getValue();
-        this.getView().callingPanel.groupId = this.lookupReference('groupField').getValue();
+        var eventType,callingPanel;
```

Comment:
Are you sure you have check formatting after fixing issue? Please double-check everything before submitting pull request.